routes/chists.rb

```diff
@@ -3,27 +3,47 @@
 module ChistApp
   class Chists < Cuba
     define do
-      on put do
-        on ":id/favorite" do |id|
-          chist = Chist[id]
-          not_found! unless chist && chist.user_id == current_user.id
-
-          current_user.toggle_favorite(chist)
-
-          if req.xhr?
-            res.write(
-              {:id => chist.id, :favorite => current_user.favorited?(chist)}.to_json
-            )
-          else
-            res.redirect("/chists/#{chist.id}")
-          end
+      on uuid_param do |chist_id|
```

Comment:
I find this hard to read. I'd prefer having something like:
```Ruby
on ":id" do |chist_id|
  not_found! unless uuid_param? && chist = Chist[chist_id]
end
```
I find this more declarative when reading routes. Thoughts?